README.md

```diff
@@ -23,8 +23,27 @@ Working with Gulp
 
 ) gulp watch
 
-Publish
-=======
- 
-) gulp publish ot create zip for the chrome store
+Gulp Tasks
+==========
 
+gulp publish
+------------
+) runs coffee
+) removes debug lines e.g. console.log
+) Creates a zip file for uploading to the chrome store
+
+gulp dev-start
+-----------
+1) Creates a manifest.json file in the project root for loading the extension unpacked.
+2) Appends development server to appropriate locations in the TEMPORARY manifest.json
+
+gulp dev-stop
+-------------
+1) Over writes the dev manifest.json for testing agains the production server
+
+gulp watch
+----------
+watches coffee files for changes
+
+
+Important NOTE:  prod directory contain the manifest document that will ship to the chrome store.
```

Comment:
The Markdown here isn't quite right - lots of stray characters and such. You can use `markdown` to preview the rendered doc. There's also a "Display the rich diff" view right here in the PR review.